omni_exchange.gemspec

```diff
@@ -12,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = 'OmniExchange converts currencies using up-to-the-minute foreign exchange rates.'
   spec.homepage      = 'https://degica.com'
   spec.license       = 'MIT'
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
+  spec.required_ruby_version = Gem::Requirement.new('>= 3.0.1')
```

Comment:
I truly believe this was just set as a default value when creating the scaffolding for the gem. There are several breaking changes between major versions and ruby 2.7.x already reached their EOL so went ahead with setting the first major stable versions from ruby 3.0

Reply:
3.0.x is also EOL, I think we can specify >= 3.1 here